agent/apiharness/src/main/java/com/intuit/tank/harness/TestPlanStarter.java

```diff
@@ -48,7 +48,7 @@ public class TestPlanStarter implements Runnable {
     private Dimension jobId;
     private boolean standalone;
     private Date send = new Date();
-    private static final int interval = 30; // SECONDS
+    private static final int interval = 15; // SECONDS
```

Comment:
@Zakaria-Kofiro Small but can we update this line as well to reflect the changed value on line 432: 
`send = DateUtils.addSeconds(new Date(), interval); // 30 SECONDS`